l2-frontend/src/construct/ConstructCompany.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <div class="main">
+    <div class="box card-1 card-no-hover">
+      <h5 class="text-center">
+        Компании
+      </h5>
+      <input
+        v-model="search"
```

Comment:
Такие поля лучше очищать от ненужных пробелов
```suggestion
        v-model.trim="search"
```